UI/DesignWnd.cpp

```diff
@@ -144,14 +144,13 @@ namespace {
             if (!exists(saved_designs_dir))
                 return;
 
-            std::map<std::string, std::unique_ptr<ShipDesign>> file_designs;
-            parse::ship_designs(saved_designs_dir, file_designs);
+            std::vector<std::unique_ptr<ShipDesign>> file_designs;
+            std::vector<boost::uuids::uuid> ordering;
+            parse::ship_designs(saved_designs_dir, file_designs, ordering);
```

Comment:
In my tests, this doesn't find any saved designs. `file_designs` remains empty (in the debugger) after the parse, despite having two saved design files in `C:\Users\Geoff\AppData\Roaming\FreeOrion\shipdesigns`. The designs list remains empty.